db/migrate/20171221044254_add_license_to_statuses.rb

```diff
@@ -0,0 +1,5 @@
+class AddLicenseToStatuses < ActiveRecord::Migration[5.1]
+  def change
+    add_column :statuses, :license_url, :text, null: true
```

Comment:
For Postgres, this is the canonical article: https://www.depesz.com/2010/03/02/charx-vs-varcharx-vs-varchar-vs-text/

I'm willing to do something else if we want to support other DBMSes more natively?

Reply:
We don't plan on supporting any other DB type since we already use some postgres-specific syntax and column types. However, no other URL has a "text" type so far and I don't see a reason to do this one differently.

Reply:
Cool -- I value consistency. I'll make the change.